src/PLLModelGetLanguagesListDynamicMethodReturnTypeExtension.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * Set return type of PLL_Model->get_languages_list().
+ */
+
+declare(strict_types=1);
+
+namespace WPSyntex\Polylang\PHPStan;
+
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\PropertyFetch;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\Type\Constant\ConstantArrayType;
+use PHPStan\Type\Constant\ConstantStringType;
+use PHPStan\Type\ArrayType;
```

Comment:
```suggestion
use PHPStan\Type\ArrayType;
use PHPStan\Type\IntersectionType;
```
`IntersectionType` is used bellow.